node/CHANGELOG.md

```diff
@@ -10,6 +10,15 @@ All notable changes to this project will be documented in this file.  The format
 [comment]: <> (Security:   in case of vulnerabilities)
 
 
+## 1.4.14
+
+### Changed
+* Update `openssl` and `openssl-sys` to latest versions.
+
+### Fixed
```

Comment:
Is this supposed to be here?

Reply:
Yeah, I forgot to add it when I opened #3750.